src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/configuration/configuration-form/configuration-form.component.ts

```diff
@@ -118,7 +123,7 @@ export class ConfigurationFormComponent extends CdForm implements OnInit {
         this.configForm.get('values').get(value.section).setValue(sectionValue);
       });
     }
-
+    this.forceUpdate = !this.response.can_update_at_runtime && response.name.includes(RGW);
```

Comment:
Perhaps add a generic helper function and move this alongwith the constant in a generic file calling at both places.
```suggestion
    this.forceUpdate = !this.response.can_update_at_runtime && hasRGW(response.name);
```